### PR TITLE
fix code style issues in m/*.py

### PR DESCRIPTION
**Pull request by Flamefire**

### CODE REVIEW

File: easybuild/easyblocks/m/modeller.py [hunk 1]
```diff
@@ -56,17 +55,18 @@ def install_step(self):
 
         # by default modeller tries to install to $HOME/bin/modeller9.13
         # get this path to use it in the question/answer
-        default_install_path = "[%s]:" % os.path.join(os.path.expanduser('~'), 'bin', 'modeller%s' % self.cfg['version'])
+        default_install_path = "[%s]:" % os.path.join(
+            os.path.expanduser('~'), 'bin', 'modeller%s' % self.cfg['version'])
```
Comment (boegel) [inline easybuild/easyblocks/m/modeller.py:59]:
```suggestion
home = os.path.expanduser('~')
default_install_path = "[%s]:" % os.path.join(home, 'bin', 'modeller%s' % self.cfg['version'])

File: easybuild/easyblocks/m/molpro.py [hunk 2]
```diff
@@ -86,9 +86,10 @@ def configure_step(self):
                 symlink(self.cfg['license_file'], self.license_token)
                 self.log.debug("Symlinked %s to %s", self.cfg['license_file'], self.license_token)
             else:
-                self.log.warning("No licence token found at either {0} or via 'license_file'".format(self.license_token))
-        
-        # Only do the rest of the configuration if we're building from source 
+                self.log.warning(
+                    "No licence token found at either {0} or via 'license_file'".format(self.license_token))
```
Comment (boegel) [inline easybuild/easyblocks/m/molpro.py:90]:
```suggestion
                self.log.warning("No licence token found at either %s or via 'license_file'",
                                 self.license_token)

**Review by wpoely86 — APPROVED** [review]

File: easybuild/easyblocks/m/mono.py [hunk 3]
```diff
@@ -140,14 +138,15 @@ def configure_step(self):
             if self.cfg['parallel']:
                 par = "-j %s" % self.cfg['parallel']
 
-            config_cmd = "%s ./configure --prefix=%s %s" % (self.cfg['preconfigopts'], tmp_mono_path, self.cfg['configopts'])
+            config_cmd = "%s ./configure --prefix=%s %s" % (
+                self.cfg['preconfigopts'], tmp_mono_path, self.cfg['configopts'])
```
Comment (boegel) [inline easybuild/easyblocks/m/mono.py:142]: Why not make it consistent with how `build_cmd` is constructed below:

```suggestion
            config_cmd = ' '.join([
                self.cfg['preconfigopts'],
                "./configure",
                "--prefix=" + tmp_mono_path,
                self.cfg['configopts'],
            ])

File: easybuild/easyblocks/m/mothur.py [hunk 4]
```diff
@@ -55,7 +55,8 @@ def guess_start_dir(self):
     def configure_step(self, cmd_prefix=''):
         """Configure Mothur build by setting make options."""
         # Fortran compiler and options
-        self.cfg.update('buildopts', 'FORTAN_COMPILER="%s" FORTRAN_FLAGS="%s"' % (os.getenv('F77'), os.getenv('FFLAGS')))
+        self.cfg.update('buildopts', 'FORTAN_COMPILER="%s" FORTRAN_FLAGS="%s"' %
+                        (os.getenv('F77'), os.getenv('FFLAGS')))
```
Comment (boegel) [inline easybuild/easyblocks/m/mothur.py:59]:
```suggestion
        self.cfg.update('buildopts', 'FORTAN_COMPILER="%s"' % os.getenv('F77'))
        self.cfg.update('buildopts', 'FORTRAN_FLAGS="%s"' % os.getenv('FFLAGS'))

File: easybuild/easyblocks/m/mtl4.py [hunk 5]
```diff
@@ -41,10 +41,10 @@ def sanity_check_step(self):
         incpref = os.path.join('include', 'boost', 'numeric')
 
         custom_paths = {
-                        'files':[],
-                        'dirs':[os.path.join(incpref, x) for x in ["itl", "linear_algebra",
-                                                                   "meta_math", "mtl"]]
-                     }
+            'files': [],
+            'dirs': [os.path.join(incpref, x) for x in ["itl", "linear_algebra",
+                                                        "meta_math", "mtl"]]
```
Comment (boegel) [inline easybuild/easyblocks/m/mtl4.py:46]:
```suggestion
            'dirs': [os.path.join(incpref, x) for x in ["itl", "linear_algebra", "meta_math", "mtl"]],

File: easybuild/easyblocks/m/mumps.py [hunk 6]
```diff
@@ -62,19 +61,20 @@ def configure_step(self):
...
             if LooseVersion(self.version) >= LooseVersion('5.0.0'):
                 make_inc_templ = 'Makefile.debian.%s'
             else:
                 make_inc_templ = 'Makefile.gfortran.%s'
 
             optf = "-DALLOW_NON_INIT"
         else:
-            raise EasyBuildError("Unknown compiler family, don't know to prepare for building with specified toolchain.")
+            raise EasyBuildError(
+                "Unknown compiler family, don't know to prepare for building with specified toolchain.")
```
Comment (boegel) [inline easybuild/easyblocks/m/mumps.py:77]:
```suggestion
            error = "Unknown compiler family, don't know to prepare for building with specified toolchain."
            raise EasyBuildError(error)

File: easybuild/easyblocks/m/mumps.py [hunk 7]
```diff
@@ -163,12 +163,11 @@ def sanity_check_step(self):
         """Custom sanity check for MUMPS."""
         custom_paths = {
             'files': ["include/%s%s.h" % (x, y) for x in ["c", "d", "s", "z"]
```
Comment (boegel) [inline easybuild/easyblocks/m/mumps.py:165]:
```suggestion
            'files': [os.path.join("include", "%s%s.h" % (x, y)) for x in ["c", "d", "s", "z"]

File: easybuild/easyblocks/m/mumps.py [hunk 8]
```diff
@@ -163,12 +163,11 @@ def sanity_check_step(self):
...
-                                                for y in ["mumps_c", "mumps_root", "mumps_struc"]] +
-                     ["include/mumps_compat.h", "include/mumps_c_types.h"] +
-                     ["lib/lib%smumps.a" % x for x in ["c", "d", "s", "z"]] +
-                     ["lib/libmumps_common.a", "lib/libpord.a"],
+                      for y in ["mumps_c", "mumps_root", "mumps_struc"]] +
+            ["include/mumps_compat.h", "include/mumps_c_types.h"] +
+            ["lib/lib%smumps.a" % x for x in ["c", "d", "s", "z"]] +
+            ["lib/libmumps_common.a", "lib/libpord.a"],
```
Comment (boegel) [inline easybuild/easyblocks/m/mumps.py:169]:
```suggestion
            [os.path.join("include", "mumps_compat.h"), os.path.join("include", "mumps_c_types.h")] +
            [os.path.join("lib", "lib%smumps.a" % x) for x in ["c", "d", "s", "z"]] +
            [os.path.join("lib", "libmumps_common.a"), os.path.join("lib", "libpord.a")],

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyblocks/m/modeller.py [hunk 9]
```diff
@@ -56,17 +55,18 @@ def install_step(self):
 
         # by default modeller tries to install to $HOME/bin/modeller9.13
         # get this path to use it in the question/answer
-        default_install_path = "[%s]:" % os.path.join(os.path.expanduser('~'), 'bin', 'modeller%s' % self.cfg['version'])
+home = os.path.expanduser('~')
```
[the remark this answers is not in the document]
Comment (boegel) [reply easybuild/easyblocks/m/modeller.py:58]: ouch, broken indent, sorry :(

File: easybuild/easyblocks/m/mumps.py (hunk 6 above)
Comment (Flamefire) [reply easybuild/easyblocks/m/mumps.py]: Done slightly differently and fixed the missing "how"

**Review by boegel — APPROVED** [review]

**Pull request merged**